packages/ui/src/components/LogtoSignature/index.module.scss

```diff
@@ -0,0 +1,48 @@
+@use '@/scss/underscore' as _;
```

Comment:
for icon filenames:

<img width="221" alt="image" src="https://user-images.githubusercontent.com/14722250/222208190-71e06256-cdc9-4577-af4b-0c0852ee9d68.png">

should we align one filename style? saw most are hyphen, and ac also uses hyphen